### ISSUE TITLE
 Text in table is white on white background

### ISSUE THREAD

**Issue by mgunyho** [arXiv/html_feedback#2423]
### Description

The text in this table is very light gray on white background, basically unreadable. I'm using Firefox 131 on Fedora 40, with the theme set to "system", which I think is the dark theme.

### (Optional:) Please add any files, screenshots, or other information here.

![image](https://github.com/user-attachments/assets/f0b6f501-dec1-418f-819f-1c8f3c5d2f62)

Here I have selected some of the text to show that it's there.

### (Required) What is this issue most closely related to? Select one.

Other

### Internal issue ID

33f975fb-3a8f-4059-bad1-38d981534a05

### Paper URL

https://arxiv.org/html/2308.02312v4

### Browser

Firefox/131.0

### Device Type

Desktop

**Comment by dginev** [comment]
Thank you for the report. I am seeing this table show normally on Firefox/131.0 today, so I am wondering if there is some additional information needed to identify the cause.

Could you share the operating system you are using for this Firefox screenshot?

![image](https://github.com/user-attachments/assets/403c7b1c-7a57-4c7d-a92a-f00f92d67568)

**Comment by mgunyho** [comment]
Did you try with dark theme enabled in the settings?

![image](https://github.com/user-attachments/assets/3fccd0ac-4fb9-4309-9e65-2cc1b37d7311)

**Comment by dginev** [comment]
Yes, exactly with the menu you are showing, this is Dark theme, Firefox 131, Ubuntu 24.04:

![image](https://github.com/user-attachments/assets/71f7d185-eb54-40d0-a0f7-974919cf18e6)

**Comment by mgunyho** [comment]
Hmm, now I checked that paper again and something seems to have changed, previously I had a white background but a dark sidebar on the left, but now the text is also on a dark background, so that is correct. The table I posted in my screenshot looks OK, but now Table 2 looks like this:

![image](https://github.com/user-attachments/assets/807f5117-3c89-4bf1-8c41-9564fe281406)

There are also some sections of text that are black on the dark background:

![image](https://github.com/user-attachments/assets/88b5696a-026b-4ee1-bc22-606edabb4c12)

**Comment by mgunyho** [comment]
I also see some formatting problems in the "Finding" boxes, but this is perhaps a separate issue not related to the dark theme:

![image](https://github.com/user-attachments/assets/a50bd290-d50c-4e73-a4ab-168f39357002)

**Comment by dginev** [comment]
@mgunyho Right, the new issues you are posting are known (and cataloged) - but it is a relief to see the new information fit my own view of the article. We can leave the issue open until the article looks better overall.

Thanks for the new information. The original glitch may have been due to a partial CSS load, I'll keep an eye out for other reports of this nature.

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
